### ISSUE TITLE
Improvement in Authentication docs for ClusterTriggerAuthentication

### ISSUE THREAD

**Issue by Shubham82** [kedacore/keda-docs#442]
In keda Authentication concept there is  a [Authentication scopes: Namespace vs. Cluster](https://keda.sh/docs/2.2/concepts/authentication/#authentication-scopes-namespace-vs-cluster) in which the following below things is  mentioned for ClusterTriggerAuthentication:
```
authenticationRef:
       name: {trigger-authentication-name}
       kind: ClusterTriggerAuthentication

```
but for name it should be either {cluster-trigger-authentication-name} or {cluster-trigger-auth-name} instead of  {trigger-authentication-name}. For better user experience and symmetry of the docs.

**Comment by tomkerkhove** [comment]
`{cluster-trigger-authencation-name}` sounds good to me, thanks!

**Comment by Shubham82** [comment]
Thanks @tomkerkhove  for the response.
I will soon create a PR for it.

**Comment by Shubham82** [comment]
Fixed it here #443
